src/ui/ResultList/ResultList.ts

```diff
@@ -237,6 +262,10 @@ export class ResultList extends Component {
       this.options.resultContainer.appendChild(resultElement);
       this.triggerNewResultDisplayed(Component.getResult(resultElement), resultElement);
     });
+    if (this.currentLayout == 'card') {
+      // Used to prevent last card from spanning the grid's whole width
+      _.times(3, () => this.options.resultContainer.appendChild($$('div').el));
```

Comment:
Hum, this means our card layout is hardcoded to 3, even if we modify the css rules ?

That's looks hackish...

If we *really* need this, we need to provide a way for users to modify the number of card in a row for their template, easily.